Automattic/wp-calypso
# WPCOM: Migrate `wpcom.undocumented()` publicize to wpcom.req

#### Changes proposed in this Pull Request

This PR migrates all the `wpcom.undocumented()` publicize methods to `wpcom.req.get()`.

Part of the ongoing effort to get rid of `wpcom.undocumented()`.

#### Testing instructions

* Go to `/marketing/connections/:site` where `:site` is one of your sites.
* Verify the currently connected social networks under "Publicize posts" are still fetched correctly.
* Try disconnecting one and verify that still works well.
* Try reconnecting and verify that still works well.
* Try expanding the accordion of a network and toggling "Connection available to all administrators, editors, and authors" and verify that still works well.
* Make sure the site has premium or a more expensive plan that supports republicize and go to `/posts/:site`, pick a post and click on the ellipsis, then click "Share". In there, click the "Share" button and verify sharing still works well.
